### DISCUSSION TITLE
What are the default Quarto mainfont, sansfont and monofont?

### DISCUSSION THREAD

**Discussion by danielvartan** [quarto-dev/quarto-cli#7479] [Q&A] [answered]
### Description

For a LaTeX format on Windows machines, what are the default Quarto mainfont, sansfont and monofont?

Are those settings described somewhere?

**Comment by mcanouil** [comment, marked as answer]
Why are you closing your questions?

Defaults comes with the template/partials used and classes, see <https://quarto.org/docs/journals/templates.html#templates>.
Quarto/Pandoc does not set LaTeX default, so you need to look into LaTeX document class, etc. to see what's the default for a given document class.

**Comment by danielvartan** [comment]
Thank you, @mcanouil.

I closed the question because I figured that was the answer.

**Reply by mcanouil** [reply]
The thread was closed before any answer and without any comments thus my question.
